lambda_powertools/prometheus.py

```diff
@@ -0,0 +1,78 @@
+import os
+import json
+import sys
+
+modulename = 'prometheus_client'
+if modulename in sys.modules:
+    raise Exception("prometheus_client already imported, lambda_powertools.prometheus must be imported before")
```

Comment:
<img width="594" alt="Screenshot 2023-11-09 at 11 32 47" src="https://github.com/spreaker/python-lambda-powertools/assets/4105325/c4bbcc08-07dc-4708-acd5-3ad390bedb41">
<img width="946" alt="Screenshot 2023-11-09 at 11 33 06" src="https://github.com/spreaker/python-lambda-powertools/assets/4105325/b576237f-efec-4a84-9c7b-2967d5b020e0">

Reply:
If possible, describe why you need this. So that I understand what you've been hitting and see if it can be handled differently in a future iteration.

_btw, if it's only related to the environment variable that I see below, the module can be reloaded_

Reply:
Because the global `REGISTRY` is instantiated at the first prometheus_client import, and the `PROMETHEUS_DISABLE_CREATED_SERIES` should be set before the first import. The problem is described here (https://github.com/prometheus/client_python/issues/933) and I see now that they just released a configuration for that (https://github.com/prometheus/client_python/releases/tag/v0.18.0). So maybe I can remove that shit 😅